ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.40f1
-m_EditorVersionWithRevision: 2019.4.40f1 (ffc62b691db5)
+m_EditorVersion: 2021.3.9f1
+m_EditorVersionWithRevision: 2021.3.9f1 (ad3870b89536)
```

Comment:
This is the incorrect version of Unity. Please use `2019.4.40f1` for this project. The level is still working in `2019.4.40f1`, so I will revert some changes on this branch and approve this pull request.